libs/libcommon/src/libcommon/queue.py

```diff
@@ -1019,6 +1046,27 @@ def _get_df(self, jobs: list[FlatJobInfo]) -> pd.DataFrame:
         # ^ does not seem optimal at all, but I get the types right
 
     def get_pending_jobs_df(self, dataset: str, job_types: Optional[list[str]] = None) -> pd.DataFrame:
+        filters = {"dataset": dataset}
+        if job_types:
+            filters["type"] = {"$in": job_types}  # type: ignore
+        df = JobDocument.fetch_as_df(query=filters)
+        df.rename(columns={"_id": "job_id"}, inplace=True)
+        df["priority"] = pd.Categorical(
```

Comment:
It is needed for https://github.com/huggingface/dataset-viewer/blob/main/libs/libcommon/src/libcommon/state.py#L53
same for "priority".